### PR TITLE
refactor(router-store): refactor router_store_module into separate files

### LINKED ISSUE #1298: Router Store: Refactor router_store_module into separate files
## If accepted, I would be willing to submit a PR for this feature

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

The actions, reducer and module are all in one file.

### PR DESCRIPTION
**Pull request by kouMatsumoto**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #1298

## What is the new behavior?
Divide code in router_store_module.ts into separate files.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

## Other information
Closes #1298 

- I alphabetized import paths.
- I ran `yarn run prettier` just after `yarn`.

Some indentations are changed, but I used prettier@1.13.7 (latest for package.json).

### CODE REVIEW

File: modules/store-devtools/spec/store.spec.ts [hunk 1]
```diff
@@ -709,7 +709,9 @@ describe('Store Devtools', () => {
     it('should add pause action', () => {
       const liftedState = fixture.getLiftedState();
       expect(liftedState.nextActionId).toBe(4);
-      expect(liftedState.actionsById[3].action.type).toEqual('@ngrx/devtools/pause');
```
Comment (brandonroberts) [inline modules/store-devtools/spec/store.spec.ts]: Revert these changes

File: modules/store-devtools/src/reducer.ts [hunk 2]
```diff
@@ -123,12 +123,12 @@ function recomputeStates(
     const entry: ComputedState = shouldSkip
       ? previousEntry
       : computeNextEntry(
-        reducer,
```
Comment (brandonroberts) [inline modules/store-devtools/src/reducer.ts]: Revert these changes

File: modules/store-devtools/spec/store.spec.ts (hunk 1 above)
Comment (kouMatsumoto) [reply modules/store-devtools/spec/store.spec.ts]: Sorry, I reverted irrelevant files

File: modules/store-devtools/src/reducer.ts (hunk 2 above)
Comment (kouMatsumoto) [reply modules/store-devtools/src/reducer.ts]: @brandonroberts 
I created a pull request to fix the indentations.
https://github.com/ngrx/platform/pull/1301
